tests/tests.py

```diff
@@ -139,9 +154,50 @@ def test_chunk_prefill(model = "mistralai/Mistral-7B-Instruct-v0.2") -> pd.DataF
     final_result = run_test_case(test_case)
     return final_result
 
+def test_chunk_and_prefix(model = "mistralai/Mistral-7B-Instruct-v0.2") -> pd.DataFrame:
+    # Start two servers with lmcache
+    config = CreateSingleLocalBootstrapConfig(8000, 0, model, "configs/lmcache_local_cpu.yaml")
+    config.vllm_optional_config["enable_chunked_prefill"] = True
+    config.vllm_optional_config["enable_prefix_caching"] = ""
+
+    # Set vllm configuration for different models
+    ModelConfig(model, config)
+
+    # Experiments: 8K, 16K, 24K shared context, each experiments has 5 queries
+    lengths = [8192, 16384, 24576]
+    experiments = [CreateDummyExperiment(5, length ) for length in lengths]
+
+    test_case = TestCase(
+            experiments = experiments,
+            engines = [config])
+
+    # Run test case
+    final_result = run_test_case(test_case)
+    return final_result
+
+def test_vary_length_workload(model = "mistralai/Mistral-7B-Instruct-v0.2") -> pd.DataFrame:
+    # Start two servers with lmcache
+    config = CreateSingleLocalBootstrapConfig(8000, 0, model, "configs/lmcache_local_cpu.yaml")
+
+    # Set vllm configuration for different models
+    ModelConfig(model, config)
+
+    # One experiments: context length sequence 12K, 24K, 24K
+    lengths = [12288, 24576, 24576]
+    experiments = [CreateVaryLengthExperiment(len(lengths), lengths )]
+
+    test_case = TestCase(
+            experiments = experiments,
+            engines = [config])
+
+    # Run test case
+    final_result = run_test_case(test_case)
+    return final_result
+
 def test_multi_turn(model = "mistralai/Mistral-7B-Instruct-v0.2") -> pd.DataFrame:
     # Start one server: with lmcache; for contrast (not saving decode KV Cache), change save_decode_cache to false
-    config = CreateSingleLocalBootstrapConfig(8000, 0, model, "configs/lmcache_local_cpu_multi.yaml")
+    config = CreateSingleLocalBootstrapConfig(8000, 0, model, "configs/lmcache_local_cpu.yaml")
+    config.vllm_optional_config["save_decode_cache"] = True
```

Comment:
Why do we put this in vllm config instead of lmcache config in yaml?

Reply:
If we put this in lmcache config in yaml, we need a config.yaml file for every E2E tests and it will result in many files; if we put this in vllm config, the tests may be considered not packed up. Which way do you think it's better?